service/docs/source/geopmsession.1.rst

```diff
@@ -27,6 +80,31 @@ core zero:
     $ echo "MSR::THERM_STATUS# core 0" | geopmsession
     0x0000000088430800
 
+This will execute one read of the signal.
+
+Reading a signal periodically
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Both a polling period and runtime must be specified.
+The polling period must be shorter than the runtime specified.
+
+A 100ms polling period with a 300ms timeout is shown below:
```

Comment:
Nit: It says "runtime" earlier and "timeout" here. I like "timeout" more since there's less ambiguity about "time spent running" vs "a software runtime" which are both relevant usages of that word in our project.